root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -1530,6 +1551,32 @@ test('Cleanup', function (t) {
             [
                 'http://en.wikipedia.org/wiki/Ramesh_Vinayakam?oldformat=true',
                 'http://en.wikipedia.org/wiki/Ramesh_Vinayakam'
+            ],
+            // BookBrainz
+            [
+                'https://bookbrainz.org/creator/8f3d202f-fa37-4b71-9e81-652db0f8b83d',
```

Comment:
Maybe add a `?test` argument or something to the URL or an `#anchor` to make sure those also get filtered out?